src/app/components/subtopics-list/search-section/search-section.component.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  Component,
+  ElementRef,
+  EventEmitter,
+  OnInit,
+  Output,
+  ViewChild,
+} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import {
+  debounceTime,
+  distinctUntilChanged,
+  filter,
+  fromEvent,
+  map,
+} from 'rxjs';
+
+@Component({
+  selector: 'app-search-section',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './search-section.component.html',
+  styleUrl: './search-section.component.scss',
+})
+export class SearchSectionComponent implements OnInit {
+  protected searchedTitle: string = '';
+  @Output() searchedValueSent = new EventEmitter<string>();
+
+  @ViewChild('subtopicsSearchValue', { static: true })
```

Comment:
static:true is by default

Reply:
Yes. I saw that from angular 8 it is but it's not working without static: true. 
![image](https://github.com/radumaftei/frontend-learning-app/assets/100227050/ed1a82b3-1df2-458d-bbba-6730a47a47ac)